extensions/wikia/DesignSystem/i18n/de.json

```diff
@@ -55,7 +55,7 @@
   "global-navigation-notifications-title": "Notifications",
   "global-navigation-search-cancel": "Cancel",
   "global-navigation-search-placeholder-fandom": "Search Stories and Wikis...",
-  "global-navigation-search-placeholder-in-wiki": "Search __sitename__",
+  "global-navigation-search-placeholder-in-wiki": "Search $1",
```

Comment:
So we are using `$1` but we have defined in `DesignSystem/i18n/build.js` map `__sitename__` to `$1`?